sdk/monitor/Azure.Monitor.OpenTelemetry.Exporter/tests/Azure.Monitor.OpenTelemetry.Exporter.Tests/StandardMetricTests.cs

```diff
@@ -0,0 +1,68 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using Azure.Monitor.OpenTelemetry.Exporter.Internals;
+using Azure.Monitor.OpenTelemetry.Exporter.Models;
+using OpenTelemetry.Metrics;
+using OpenTelemetry;
+using Xunit;
+using Azure.Monitor.OpenTelemetry.Exporter.Tests.CommonTestFramework;
+using OpenTelemetry.Trace;
+using System.Collections.Concurrent;
+using System.Diagnostics;
+using System.Linq;
+
+namespace Azure.Monitor.OpenTelemetry.Exporter.Tests
+{
+    public class StandardMetricTests
+    {
+        [Fact]
+        public void ValidateRequestDurationMetric()
+        {
+            var activitySource = new ActivitySource(nameof(StandardMetricTests.ValidateRequestDurationMetric));
+            var traceTelemetryItems = new ConcurrentBag<TelemetryItem>();
+            var metricTelemetryItems = new ConcurrentBag<TelemetryItem>();
+            var exporterOptions = new AzureMonitorExporterOptions();
+            exporterOptions.ConnectionString = "InstrumentationKey=Ikey;IngestionEndpoint=https://westus2-0.in.applicationinsights.azure.com/;LiveEndpoint=https://westus2.livediagnostics.monitor.azure.com/";
```

Comment:
nit: To keep this simple, I think you can remove the IngestionEndpoint and LiveEndpoint